whois-commons/src/main/java/net/ripe/db/whois/common/domain/Maintainers.java

```diff
@@ -68,6 +68,10 @@ public Maintainers(
         return dbmMaintainers;
     }
 
+    public boolean isEnduserMaintainer(Set<CIString> mntner) {
+        return !Sets.intersection(enduserMaintainers, mntner).isEmpty();
+    }
+
     public boolean isRsMaintainer(CIString mntner) {
         return isRsMaintainer(Sets.newHashSet(mntner));
```

Comment:
Collections.singleton(mntner) is faster than creating a new Set with one element